common/riff/parent_chunk_data.ksy

```diff
@@ -0,0 +1,13 @@
+meta:
+  id: parent_chunk_data
```

Comment:
Same thing: this RIFF-specific piece will become repo-wide `ParentChunkData` after compilation.